app/src/main/java/ua/test/mega/tester/rest/NotificationRestController.java

```diff
@@ -18,47 +17,49 @@
+        this.notificationProcessor = notificationProcessor;
+    }
+
+    @GetMapping(path = "/", produces = MediaType.TEXT_EVENT_STREAM_VALUE)
+    public Flux<NotificationDTO> getNotificationsForLoggedInUser() {
 
-	@Autowired
-	public NotificationRestController(NotificationProcessor notificationProcessor) {
-		this.notificationProcessor = notificationProcessor;
-	}
+        return notificationProcessor.provideNotificationsForLoggedinUser()
+                .map(this::toNotificationDTO);
+    }
 
-	@GetMapping(path = "/", produces = MediaType.TEXT_EVENT_STREAM_VALUE)
-	public Flux<NotificationDTO> getNotificationsForLoggedInUser() {
-		return notificationProcessor.provideNotificationsForLoggedinUser()
-				.map(this::toNotificationDTO);
-	}
+    @Secured("ROLE_ADMIN")
+    @GetMapping(path = "/all", produces = MediaType.TEXT_EVENT_STREAM_VALUE)
+    public Flux<NotificationDTO> getAllNotifications() {
 
-	@Secured("ROLE_ADMIN")
-	@GetMapping(path = "/all", produces = MediaType.TEXT_EVENT_STREAM_VALUE)
-	public Flux<NotificationDTO> getAllNotifications() {
-		return notificationProcessor.provideAllNotifications()
-				.map(this::toNotificationDTO);
-	}
+        return notificationProcessor.provideAllNotifications()
+                .map(this::toNotificationDTO);
+    }
 
-	private NotificationDTO toNotificationDTO(Notification notification) {
-		return NotificationDTO.builder()
-				.message(notification.getMessage())
-				.type(toNotificationTypeDTO(notification.getType()))
-				.userId(notification.getUserId())
-				.error(notification.getError())
-				.build();
-	}
+    private NotificationDTO toNotificationDTO(Notification notification) {
+        return NotificationDTO.builder()
+                .message(notification.getMessage())
+                .type(toNotificationTypeDTO(notification.getType()))
+                .userId(notification.getUserId())
+                .error(notification.getError())
+                .build();
+    }
 
-	private NotificationTypeDTO toNotificationTypeDTO(NotificationType type) {
-		switch (type) {
-			case USER:
-				return NotificationTypeDTO.USER;
-			case CONFIGURATION:
-				return NotificationTypeDTO.CONFIGURATION;
-			case ADMINISTRATION:
-				return NotificationTypeDTO.ADMINISTRATION;
-			case INFO:
-				return NotificationTypeDTO.INFO;
-			default:
-				throw new IllegalArgumentException("Unknown NotificationType " + type);
-		}
-	}
+    private NotificationTypeDTO toNotificationTypeDTO(NotificationType type) {
+        switch (type) {
+            case USER:
+                return NotificationTypeDTO.USER;
+            case CONFIGURATION:
+                return NotificationTypeDTO.CONFIGURATION;
+            case ADMINISTRATION:
+                return NotificationTypeDTO.ADMINISTRATION;
+            case INFO:
+                return NotificationTypeDTO.INFO;
+            default:
+                throw new IllegalArgumentException("Unknown NotificationType " + type);
+        }
+    }
 }
```

Comment:
As you can see IT is impossible to review.
Please revert changes

Reply:
done